react/week1/todo_list/src/ListItem.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+function ListItem(props) {
```

Comment:
When passing props, you can actually use a very cool javascript feature called destructuring. For example

    const ListItems = ({description, deadline}) => { }

Then you can access the variables without using `props.`